inc/class-saml.php

```diff
@@ -646,6 +669,15 @@ public function handleLoginAttempt( $net_id, $email ) {
 		}
 	}
 
+	private function logData( $key, $data ) {
+		if ( ! is_null( $this->log ) ) {
```

Comment:
If environment variables are not present, then log property is null and any log is sent.